students/vvinodh/Lesson5/mailroom.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+from operator import itemgetter
+import os
+import datetime
+
+
+def menu():
+
+    print("""\n------------------ MENU ------------------\n
+PLEASE CHOOSE FROM THE FOLLOWING THREE OPTIONS
+1. Send Thank You
+2. Export Letters to Everyone
+3. Create a Report
+4. Quit\n""")
+
+    choice = input("-> ").lower()[0]
+    options.get(choice, menu)()
+
+
+def thankyou():
+
+    print("""\n------------------ THANK YOU ------------------\n
+type 'list' to to see a complete list of donors -
+type 'menu' at any time to return to the menu""")
+
+    # asks for a name from the user, or for the 'list' prompt
+    name_check = input("Enter first and last name of a donor\n\n-> ").lower()
+
+    try:
+        options.get(name_check)()
+    except TypeError:
```

Comment:
This outer try/except does not seem necessary as you would not generally place the expected flow of the program inside of an `except` block